neutron_plugin/router.py

```diff
@@ -87,69 +95,76 @@ def create(neutron_client, args, **kwargs):
     set_neutron_runtime_properties(ctx, r, ROUTER_OPENSTACK_TYPE)
 
 
+# @operation
```

Comment:
Why is this commented out?

Reply:
@EarthmanT  This is no longer used anymore and It should be used when add support to update router operation

Reply:
OK, I think to avoid affecting present users (one customer I know of in particular), we should adjust this to call `update_router` with a deprecation message. What do you say?